topics/sequence-analysis/tutorials/annotation-with-prokka/tours/tour.yaml

```diff
@@ -1 +1,195 @@
+      process for the annotation of protein coding regions: first, protein
+      coding regions on the genome are identified using <a
+      href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2848648/">Prodigal</a>;
+      second, the function of the encoded protein is predicted by similarity to
+      proteins in one of many protein or protein domain databases. Prokka is a
+      software tool that can be used to annotate bacterial, archaeal and viral
+      genomes quickly, generating standard output files in GenBank, EMBL and gff
+      formats. More information about Prokka can be found <a
+      href="https://github.com/tseemann/prokka">here</a>.
+    backdrop: true
+  - title: History options
+    element: '#history-options-button'
+    content: >-
+      We will start the analyses by creating a new history. Click on this button
+      and then "Create New"
+    placement: left
+  - title: Obtaining our data
+    content: >-
+      Prokka requires assembled contigs, available on <a
+      href="https://zenodo.org/record/1156405">zenodo</a>.
+    backdrop: true
+  - title: Uploading the new data
+    element: '#tool-panel-upload-button .fa.fa-upload'
+    content: We need to upload data. Open the Galaxy Upload Manager
+    placement: right
+    postclick:
+      - '#tool-panel-upload-button .fa.fa-upload'
+      - '#btn-reset'
+  - title: Uploading the input data
+    element: '#btn-new'
+    content: Click on Paste/Fetch Data
+    placement: right
+    postclick:
+      - '#btn-new'
+  - title: Uploading the input data
+    element: .upload-text-column .upload-text .upload-text-content.form-control
+    content: >-
+      Insert the links here.<br> Load the file into your history from <a
+      href="https://zenodo.org/record/1156405/files/contigs.fasta5">zenodo</a>.
+    placement: right
+  - title: Uploading the input data
+    element: '#btn-start'
+    content: Click on "Start" to start loading the data to history
+    placement: right
+    postclick:
+      - '#btn-start'
+  - title: Uploading the input data
+    element: '#btn-close'
+    content: >-
+      The upload may take a while.<br> Hit the close button to close this
+      window.
+    placement: right
+    postclick:
+      - '#btn-close'
+  - title: Rename the input data
+    element: '.history-right-panel .list-items > *:first'
+    content: >-
+      The uploaded data is in the history, but its name corresponds to the link.
+      We want to rename them it to something meaningful, e.g.
+      contigs.fasta<br><br> <ul>
+        <li>Click on the pencil icon beside the file to "Edit Attributes"</li>
+        <li>Change the name in "Name" to get only the name of the sample</li>
+      </ul>
+    position: left
+  - title: Annotate the genome
+    content: Now we will run the tool called Prokka.
+    backdrop: true
+  - title: Annotate the genome
+    element: '#tool-search-query'
+    content: Search for "Prokka" tool
```

Comment:
Type in "Prokka" and press "Enter". Click on the tool "Prokka" in the tool list.